src/component/RestaurantButton.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+function RestaurantButton({ onClick }) {
```

Comment:
```suggestion
function RestaurantButton({ handleClick }) {
```
Usually a convention to call this `handleClick` - the name of the function.